src/rand.jl

```diff
@@ -0,0 +1 @@
+Base.rand(μ::AbstractMeasure) = rand(Random.GLOBAL_RNG, μ)
```

Comment:
Perhaps a bit nicer this way

Reply:
Absolutely,! Oh, I guess we can just have this in one place, and then for a particular measure just define `rand(::AbstractRNG, μ)`. So we won't have to have quite as much duplication as I had thought. Nice :)